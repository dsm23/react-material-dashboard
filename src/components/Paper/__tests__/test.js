import React from 'react';
import { render } from 'react-testing-library';

import Paper from '..';

describe('Paper - component', () => {
  it('renders children', () => {
    const children = 'some text';

    const { getByText } = render(<Paper>{children}</Paper>);
    expect(getByText(children)).toBeTruthy();
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Paper />);
    expect(asFragment()).toMatchSnapshot();
  });
});
