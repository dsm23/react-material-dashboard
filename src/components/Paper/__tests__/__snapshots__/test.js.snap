// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper - component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root-4 MuiPaper-elevation0-6 MuiPaper-rounded-5 CustomPaper-root-1 CustomPaper-outlined-3"
  />
</DocumentFragment>
`;
